pvr.iptvsimple/resources/language/resource.language.it_it/strings.po

```diff
@@ -78,7 +78,7 @@ msgstr "Loghi canali"
 
 msgctxt "#30031"
 msgid "Channels Logos Folder"
-msgstr "Cartella  loghi canali"
+msgstr "Cartella loghi canali"
```

Comment:
Please remove this change. We only every update the en_gb strings.po file. All other translation files are done by transifix.

Reply:
Resolved.